### ISSUE TITLE
creation of instances from abstract class

### ISSUE THREAD

**Issue by s7oev** [abaplint/transpiler#359]
The code below defines an abstract class (`vehicle`), then generates an instance from this class, and calls a setter & a getter for its `speed` attribute. Executing it in the transpiler leads to an output of `8` (the value of the instance's `speed` attribute) - i.e., the instance was successfully created, despite the fact that its class is abstract, so instance creation from it must not be allowed.

```abap
CLASS vehicle DEFINITION ABSTRACT.
  PUBLIC SECTION.
    METHODS:
      drive ABSTRACT,
      set_speed
        IMPORTING speed TYPE i,
      get_speed
        RETURNING VALUE(result) TYPE i.

  PROTECTED SECTION.
    DATA:
      speed TYPE i.
ENDCLASS.


CLASS vehicle IMPLEMENTATION.
  METHOD set_speed.
    me->speed = speed.
  ENDMETHOD.

  METHOD get_speed.
    result = me->speed.
  ENDMETHOD.
ENDCLASS.


DATA my_car TYPE REF TO vehicle.
CREATE OBJECT my_car TYPE vehicle.

my_car->set_speed( 8 ).
WRITE my_car->get_speed( ).
```

**Comment by larshp** [comment]
yea, its more a abaplint issue, it should give a syntax error

from a transpiler perspective the generated code is okay, just remove the abstract keyword, assuming the syntax is correct

**Comment by s7oev** [comment]
Yes, this is true, it should be a syntax error because of the **abstract** keyword (so linter thing), without it it would be valid code.

**Comment by larshp** [comment]
@s7oev try again, 1.2.25 published

**Comment by s7oev** [comment]
I tested with the transpiler (where nothing happens, unlike before), and the linter (where I get "_/zfoobar.prog.abap[29, 1]: vehicle is abstract, cannot be instantiated(check_syntax)_"), so it seems perfect 👍
